src/Backups/BackupsWorker.cpp

```diff
@@ -876,6 +877,7 @@ OperationID BackupsWorker::startRestoring(const ASTPtr & query, ContextMutablePt
                  on_exception,
                  process_list_element_holder = process_list_element ? process_list_element->getProcessListEntry() : nullptr]
                 {
+                    CurrentThread::QueryScope query_scope(context_in_use);
```

Comment:
I'm not sure if it's better to make query scope inside `try` or not.

Reply:
I can put it inside try just to be safe